proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/rules/TestErrorPageExceptionMapperRule.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.ida.notification.apprule.rules;
+
+import io.dropwizard.testing.junit.ResourceTestRule;
+import org.glassfish.jersey.client.ClientProperties;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+import uk.gov.ida.notification.exceptions.mappers.ErrorPageExceptionMapper;
+import uk.gov.ida.notification.exceptions.mappers.GenericExceptionMapper;
+
+import javax.ws.rs.core.Response;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestErrorPageExceptionMapperRule {
+
+    private static final String EIDAS_ERROR_PAGE_URI = "http://eidas_error_page_uri";
+    private static final String GENERIC_ERROR_PAGE_URI = "http://generic_error_page_uri";
+
+    @ClassRule
+    public static final ResourceTestRule resources = ResourceTestRule.builder()
```

Comment:
[By convention](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html), in Java, `static final` variables have ALL_CAPS names

Reply:
I followed the convention in all the other @ClassRules (EidasAuthnRequestAppRuleTests.java for example).  I feel like it's okay to not do it in this case as it's not really a 'constant' as such but a thing that's required for all the @ClassRule magic etc. and I'm pretty sure that things like the baseUri() for some of those class rules changes (the port being randomly assigned on spin-up) so ALL_CAPS could be misleading as it implies something will always be the same.